offsets.py

```diff
@@ -99,7 +99,7 @@ class OffsetsFirmware_900:
     JMP_R14 = 0xffffffff82b85693
 
 
-# FW 9.03
+# FW 9.03/9.04
 class OffsetsFirmware_903:
```

Comment:
Call this OffsetsFirmware_903_94

Reply:
OffsetsFirmware_903_94 or OffsetsFirmware_903_04?